pyanaconda/ui/gui/spokes/__init__.py

```diff
@@ -71,3 +77,22 @@ def on_back_clicked(self, button):
         # Notify the hub that we're finished.
         # The hub will be the current-action of the main window.
         self.main_window.current_action.spoke_done(self)
+
+    def clear_info(self):
+        """Clear the last set warning message and call the ancestors method."""
+        self._current_warning_message = ""
+        super().clear_info()
+
+    def show_warning_message(self, message):
+        """Show error message in the status bar.
+
+        As set_warning() animates the error bar only set new message
+        when it is different from the current one.
+        """
+        if not message:
+            self._current_warning_message = ""
```

Comment:
Maybe this is not needed if the message is set to "" in the clear_info, but I don't care much.

Reply:
Yeah, let's drop it there.